resources/win/atom.cmd

```diff
@@ -11,13 +12,40 @@ FOR %%a IN (%*) DO (
   IF /I "%%a"=="--test"       SET EXPECT_OUTPUT=YES
   IF /I "%%a"=="-v"           SET EXPECT_OUTPUT=YES
   IF /I "%%a"=="--version"    SET EXPECT_OUTPUT=YES
-  IF /I "%%a"=="-w"           SET EXPECT_OUTPUT=YES
-  IF /I "%%a"=="--wait"       SET EXPECT_OUTPUT=YES
+  IF /I "%%a"=="-w"           (
+    SET EXPECT_OUTPUT=YES
+    SET WAIT=YES
+  )
+  IF /I "%%a"=="--wait"       (
+    SET EXPECT_OUTPUT=YES
+    SET WAIT=YES
+  )
 )
 
+rem Getting the process ID in cmd of the current cmd process: http://superuser.com/questions/881789/identify-and-kill-batch-script-started-before
+set T=%TEMP%\sthUnique.tmp
```

Comment:
You make a good point - I was thinking it was inconsequential because you wouldn't be doing atom --wait multiple times simultaneously and not worth worrying about it, but with a generic name, there could be another script doing something similar.  Addressing.